docs/index.md

```diff
@@ -1,42 +1,13 @@
-Installation Guides
-===================
+Open Science Grid Site Documentation
+====================================
 
-The Open Science Grid consists primarily of a fabric of services at
-participating sites.
+!!! note
+    This is currently a work in progress as we migrate our TWiki documentation to GitHub, so please let us know of any issues that you encounter by [opening a ticket](https://ticket.opensciencegrid.org) or e-mailing [goc@opensciencegrid.org](goc@opensciencegrid.org) with a description of your issue.
 
-One of the most common ways of participating in the OSG is to install
-one of our software services at your site and provide computational
-power opportunistically to the grid.
+Welcome to the home page of the Open Science Grid (OSG) Site documentation! The documentation contained here is designed for system adminstrators that provide resources to the OSG. If you're a new resource provider, the [Installation Overview](install-overview) is a good place to get started. Otherwise, you can find a list of our documents via the navigation and search bars to the left.
```

Comment:
Let’s aim to be consistent in our terminology. Are they system administrators or site administrators? Also, this whole paragraph is very chatty — can you tighten and focus it?